test_vm/tests/suite/authenticate_message_test.rs

```diff
@@ -1,10 +1,11 @@
 use fil_actors_integration_tests::tests::account_authenticate_message_test;
 use fil_actors_runtime::test_blockstores::MemoryBlockstore;
+use std::rc::Rc;
 use test_vm::TestVM;
 
 #[test]
 fn account_authenticate_message() {
-    let store = MemoryBlockstore::new();
-    let v = TestVM::new_with_singletons(&store);
+    let store = Rc::new(MemoryBlockstore::new());
+    let v = TestVM::new_with_singletons(store);
```

Comment:
```suggestion
    let store = MemoryBlockstore::new();
    let v = TestVM::new_with_singletons(store);
```

Given the changes to the `new_with_singletons` signature, we shouldn't need to manually wrap in an `Rc`. Apply below as well.

Reply:
Fixed here and elsewhere.